internal/pkg/workspace/workspace.go

```diff
@@ -455,7 +455,8 @@ func (ws *Workspace) ListDockerfiles() ([]string, error) {
 		// Add sub-directories containing a Dockerfile one level below current directory.
 		subFiles, err := ws.fsUtils.ReadDir(wdFile.Name())
 		if err != nil {
-			return nil, fmt.Errorf("read directory: %w", err)
+			// swallow errors for unreadable directories
+			continue
```

Comment:
Do we need to update unit tests for this change?

Reply:
There are currently no unit tests reliant on it, apparently. I'll add one in my next PR.

Reply:
It turns out that afero does not check permissions when opening files. See https://github.com/spf13/afero/issues/150 for others complaining about this.